Plutarch/FFI.hs

```diff
@@ -31,47 +31,11 @@ import Generics.SOP.Type.Metadata (
   ConstructorName,
   DatatypeInfo (ADT, Newtype),
  )
-import Plutarch.Builtin (PData)
-import Plutarch.Builtin.Bool (PBool)
-import Plutarch.Builtin.Integer (PInteger)
-import Plutarch.ByteString (PByteString)
-import Plutarch.Internal.Eq (PEq, (#==))
-import Plutarch.Internal.Generic (PCode)
-import Plutarch.Internal.Newtype (PlutusTypeNewtype)
-import Plutarch.Internal.Other (POpaque, pto)
-import Plutarch.Internal.PLam (plam)
-import Plutarch.Internal.PlutusType (
-  DPTStrat,
-  DerivePlutusType,
-  PlutusType (PInner, pcon', pmatch'),
-  pcon,
-  pmatch,
- )
-import Plutarch.Internal.Quantification (PForall (PForall))
-import Plutarch.Internal.ScottEncoding (PlutusTypeScott)
-import Plutarch.Internal.Term (
-  ClosedTerm,
-  Config,
-  PDelayed,
-  PType,
-  RawTerm (RCompiled),
-  S,
-  Term (Term),
-  TermResult (TermResult),
-  compile,
-  pdelay,
-  pforce,
-  phoistAcyclic,
-  (#),
-  (:-->),
- )
-import Plutarch.Internal.Witness (witness)
-import Plutarch.List (PList, PListLike (PElemConstraint, pcons, pelimList, pnil), pconvertLists, plistEquals)
+
```

Comment:
Does anything actually _use_ the FFI module?

Reply:
Not really. might be nice to deprecate this, but not on this PR

Reply:
Will make a spinoff ticket then.